src/MIDebugEngine/Engine.Impl/DebuggedThread.cs

```diff
@@ -226,6 +226,18 @@ private ThreadContext CreateContext(TupleValue frame)
             ulong? pc = frame.TryFindAddr("addr");
             MITextPosition textPosition = MITextPosition.TryParse(frame);
             string func = frame.TryFindString("func");
+
+            if (this._debugger.MICommandFactory.Mode == MIMode.Lldb)
+            {
+                if (func.Contains("("))
+                {
+                    // LLDB adds the types of variables to the func. This doesn't conform 
+                    // the mi protocol and results in duplicate data when the ad7 stackframe
+                    // object does the same thing. 
+                    func = func.Substring(0, func.IndexOf('('));
```

Comment:
Using the Microsoft C++ compiler at least, template arguments for function types can have '(' in them. Example:

 	natcon.exe!std::_Invoker_functor::_Call<int (__cdecl*&)(int),int>(int(*)(int) & _Obj, int && <_Args_0>) Line 1375	C++
 	natcon.exe!std::invoke<int (__cdecl*&)(int),int>(int(*)(int) & _Obj, int && <_Args_0>) Line 1443	C++
 	natcon.exe!std::_Invoke_ret<int,int (__cdecl*&)(int),int>(std::_Forced<int,0> __formal, int(*)(int) & <_Vals_0>, int && <_Vals_1>) Line 1468	C++
	natcon.exe!std::_Func_impl<int (__cdecl*)(int),std::allocator<int>,int,int>::_Do_call(int && <_Args_0>) Line 212	C++